Gruntfile.js

```diff
@@ -36,6 +36,21 @@ module.exports = function(grunt){
                 src: ['src/wrap_header.js', 'src/base64.js', 'src/sha1.js', 'src/md5.js', 'src/utils.js', 'src/polyfills.js', 'src/core.js', 'src/bosh.js', 'src/websocket.js', 'src/wrapper.js', 'src/wrap_footer.js'],
                 dest: '<%= pkg.name %>'
             },
+            light: {
+                src: ['src/wrap_header.js', 'src/sha1.js', 'src/md5.js', 'src/utils.js', 'src/core.js', 'src/bosh.js', 'src/websocket.js', 'src/wrapper.js', 'src/wrap_footer.js'],
+                dest: 'strophe.light.js',
```

Comment:
Can you please add a comment here explaining what the purpose is of the `strophe.light.js` build?